src/Components/NotesPage/NotesPage.jsx

```diff
@@ -1,11 +1,62 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useArchive, useAuth, useNotes } from '../../context/';
+import { formatDate } from '../../backend/utils/authUtils';
 
-function NotesPage() {
+const NotesPage = () => {
+	const { user, encodedToken } = useAuth();
+	const { notes, postNotes, getNotes } = useNotes();
+	const { notesToArchive } = useArchive();
+	const [textField, setTextField] = useState({ title: '', body: '' });
+	const submitNote = () => {
+		postNotes({
+			title: textField.title,
```

Comment:
Before posting please make a check to see if the title and body are empty or not as applicable in your case.

Reply:
This will be done in the next PR.